adhocracy4/static/Modal.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import django from 'django'
 
 const Modal = (props) => {
-  const dismiss = props.dismissOnSubmit ? 'modal' : 'false'
+  const dismiss = props.dismissOnSubmit !== undefined ? (props.dismissOnSubmit ? 'modal' : 'false') : 'modal'
```

Comment:
While this works (and I'm generally fine with merging it) this seems a bit complicated to read to me, I would suggest to either pass `dismissOnSubmit` on the comment delete modal (no change necessary here then) or we change the prop from `dismissOnSubmit` to something like `keepOpenOnSubmit`, so the default behavior is `modal` and we only need to set this prop on modals which need non-default behavior, what do you think?